src/webgpu/shader/execution/expression/call/builtin/textureLoad.spec.ts

```diff
@@ -376,6 +376,7 @@ Parameters:
     t.skipIfTextureFormatNotSupported(format);
     t.skipIfTextureLoadNotSupportedForTextureType(texture_type);
     t.skipIfMultisampleNotSupportedForFormat(format);
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(format);
```

Comment:
This check seems, at least by name, strange. This test isn't supposed to be rendering to  the format. If it is, there's a bug or a workaround that needs to be added. Otherwise it won't test that `textureLoad` works with this format.

Reply:
It's that multisampled texture requires `GPUTextureUsage.RENDER_ATTACHMENT`. Validation fails texture with such usage when texture in not color renderable.

The usage is added: https://github.com/gpuweb/cts/blob/main/src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts#L3152

Reply:
This still seems like the wrong check. `t.skiptIfMultisampleNotSupportForFormat` should be enough because there's no such thing as a non-renderable multisample texture since the only way to use them is to render to them. It seems like `skipIfMultisampleNotSupported` should check this condition. If you can't render to it then multisample is not supported.

Reply:
You are right.

Fixed by:
* rename `skipIfMultisampleNotSupportedForFormat` -> `skipIfMultisampleNotSupportedForFormatOrSelectDevice`
  * select device or skip to ensure the format is colorRenderable
* Fix callsites: Remove `selectDeviceForRenderableColorFormatOrSkipTestCase` calls where `skipIfMultisampleNotSupportedForFormatOrSelectDevice` is already called
* Fixes for previous commits
  * remove redundant `GPUTestSubcaseBatchState.skipIfColorRenderableNotSupportedForFormat` and `GPUTestBase.skipIfMultisampleNotSupportedForFormatOrSelectDevice`
  * Fix `skipIfColorRenderableNotSupportedForFormat` by checking device.features